### DISCUSSION TITLE
Css is not working

### DISCUSSION THREAD

**Discussion by JacquesvanWyk** [jeffreyvr/dropblockeditor#20] [General]
Hey I did setup as on docs. i have on site tailwind setup but the block dont seem to show css. Did I setup something wrong? 

`

    'include_js' => true,

    'include_css' => true,

    'preview_css' => null,

`
My config is basically same as per docs

**Comment by abishekrsrikaanth** [comment]
I have the same issue as well, probably because the blocks are loaded within an iframe and the css isn't being loaded or injected into it.

**Comment by abishekrsrikaanth** [comment]
The only way I was able to get the blocks working was by updating `preview_css` to following
This is after I run `bun run build` and then reference the `app.css` from the build directory
```php
    'preview_css' => '/build/assets/app-bd4d6956.css',
```

**Reply by jeffreyvr** [reply]
This is the way to inject CSS to the iframe of the editor :)

**Reply by abishekrsrikaanth** [reply]
Thanks, i hope there is an easier way to do this so we can inject them at runtime instead of at build. Do you have an example of how to do this at runtime? Because tailwind compiles the styles based on the content of the views folder. So, we need the styles bound at runtime. I was able to get this to work https://github.com/jeffreyvr/dropblockeditor/discussions/20#discussioncomment-7041831

**Comment by abishekrsrikaanth** [comment]
Another option that helped was to include the css into the blade file inside the blocks `render` method

```php

  public function render()
    {
        return <<<'blade'
            @vite('resources/css/app.css')
                 <div class="bg-white">
        blade;
    }
```

**Reply by abishekrsrikaanth** [reply]
This is going to duplicate the styles for each block. Not sure how the final render works.
